pkg/apis/servicecatalog/types.go

```diff
@@ -226,8 +226,41 @@ type ServiceClass struct {
 	AlphaRequires []string
 }
 
+type Plan struct {
+	// PlanRef is a reference to the full plan details.
+	PlanRef v1.LocalObjectReference `json:"planRef"`
```

Comment:
I don't think it's going to be profitable to store this information in two places - they will get out of sync.